addons/l10n_be_hr_payroll/wizard/hr_individual_account_employees.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import datetime
+
+from odoo import api, fields, models
+from odoo.exceptions import UserError
+
+class IndividualAccountEmployee(models.TransientModel):
+
+    _name = 'hr.individual.account.employee'
+    _description = 'HR Individual Account Report By Employee'
+
+    year = fields.Selection([[str(i), i] for i in range(1990, datetime.datetime.now().year + 1)], string='Year', required=True,
+        default=lambda *a: str(datetime.datetime.now().year))
+    emp = fields.Many2many('hr.employee', string='Employee(s)')
+
+    @api.multi
+    def print_report(self):
+        self.ensure_one()
+        [data] = self.read()
+        data['emp'] = self.env.context.get('active_ids', [])
+        data['year'] = int(self.year)
+        employees = self.env['hr.employee'].browse(data['emp'])
+        datas = {
+            'ids': [],
+            'model': 'hr.employee',
+            'form': data
+        }
+        return self.env.ref('l10n_be_hr_payroll.action_report_individual_account').report_action(employees, data=datas)
```

Comment:
If you look at the method ``report_action`` 
It's not mandatory to provide a browse record. In fact we even do ``doc_ids.ids`` if it's the case.
```python
        if docids:
            if isinstance(docids, models.Model):
                active_ids = docids.ids
            elif isinstance(docids, int):
                active_ids = [docids]
            elif isinstance(docids, list):
                active_ids = docids
```
So you could avoid to browse the employees and pass the ids directly.